packages/calypso-products/src/plans-list.js

```diff
@@ -848,14 +852,12 @@ const getPlanJetpackSecurityT1Details = () => ( {
 	availableFor: ( plan ) => [ PLAN_JETPACK_FREE, ...JETPACK_LEGACY_PLANS ].includes( plan ),
 	getDescription: () =>
 		translate(
-			'Get next-level protection with real-time backups, real-time scan and all essential security tools.'
+			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Jetpack Security provides easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/tiered-product-cards-copy)